### ISSUE TITLE
Documentation: EV example load oddly scaled

### ISSUE THREAD

**Issue by bpalmintier** [GMLC-TDC/HELICS#2666]
**Describe the bug**
In the documentation, the multi-port EV "advanced example" has unreasonable peak values, >20MW for 5 EVs, even 

**What is the expected behavior?**
Consider that Tesla DCFC chargers are often around 100-200kW and the faster side of EV car max charge rates is ~300kW then even if all 5 ports were at max the peak would be around 1.5MW (about 10x less). Moreover such charging sessions also are simulated to last multiple days, which would not be common with anything close to DCFC. It might be relevant for Level1, but then the charge rates are around 1.5kW/charger. Certainly higher charge rates might be seen with M/HDV, but this doesn't seem to the the case in the description

**To Reproduce**
Review the output figures such as at https://docs.helics.org/en/latest/user-guide/examples/advanced_examples/advanced_brokers_multicomputer.html

**Environment (please complete the following information):**

- On-line documentation

**Additional context and information**
(e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

Generally love these examples, it would also help to somewhat increase the domain story of what is going on. In this case it is unclear how the batteries loose charge. Are they being driven? V2G, etc. And also why they seem to almost always be connected for over a week (168hr) which is unexpected since even at Level-1, folks would likely be charging only when at home and have extended times away. Even more so for DCFC where sessions might only last 20min and would fill up enough for a week or more of around town driving.

**Comment by trevorhardy** [comment]
I recommend moving this to the HELICS-Examples repo.

Can you clarify why you consider this a "bug"? Is it the lack of realism?

**Comment by bpalmintier** [comment]
Yes the "bug" (agree awkward but didn't seem to fit other categories, perhaps we need a documentation update template?) is that the data is unrealistic enough to potentially undermine HELICS credibility. For reference, even with ***@***.*** (which is on the high side) 5 EV cars would still only peak around 1MW if all simultaneously at max (unlikely) and that would only be for say 20min. For Level-2 that might be around 10kW/plug which would mean 50kW and then might last 5-10hrs, but still not for many days. In contrast the current results peak >20MW  (20x too high) and never dip below ~3MW  for a full week (60x too high, and much too long).

I had included here since this was the repo for documentation, if moving to the examples repo, maybe we need a linked one here to update the docs once fixed?

Here is the figure in question:
[cid:2d9e5da3-d402-43af-a85a-1a46357a2c98]
________________________________
